Terminal.Gui/Drivers/ConsoleDriver.cs

```diff
@@ -10,6 +10,7 @@
 using Mono.Terminal;
 using NStack;
 using Unix.Terminal;
+using System.Drawing;
```

Comment:
System.Drawing will be problematic, I would rather avoid bringing another assembly dependency.

Reply:
Sorry. I forgot remove it. Now, System.Drawing not used